custom_components/sunlit/sensor.py

```
@@ -65,143 +66,199 @@ async def async_setup_entry(
+                                family_id=family_coordinator.family_id,
+                                family_name=family_coordinator.family_name,
+                            )
+                            # Set icon if available
+                            icon = get_icon_for_sensor(key)
+                            if icon:
+                                sensor._attr_icon = icon
+                            sensors.append(sensor)
+
+                # Create individual device sensors from device coordinator
+                if device_coordinator.data and "devices" in device_coordinator.data:
+                    for device_id, device_data in device_coordinator.data[
```

Ruff (0.16.1)

[warning] Loop control variable `device_data` not used within loop body

Rename unused `device_data` to `_device_data`

(B007)